datareservoirio/storage/storage.py

```diff
@@ -297,31 +297,27 @@ def _blob_to_df(blob_url):
     with requests.Session() as session:
         retries = requests.adapters.Retry(total=5, backoff_factor=0.4, backoff_max=10)
         session.mount("https://", requests.adapters.HTTPAdapter(max_retries=retries))
-        response = session.request(method="get", url=blob_url, stream=True, timeout=30)

```

Comment:
Notes to self, trying to understand what is changed. 
The stream=True in the response returns the response.content as a stream. 
When we remove this, we read all the data into memory.